src/coll/options.rs

```diff
@@ -509,7 +509,7 @@ pub struct CountOptions {
     ///
     /// This options maps to the `maxTimeMS` MongoDB query option, so the duration will be sent
     /// across the wire as an integer number of milliseconds.
-    #[serde(deserialize_with = "deserialize_duration_from_u64_millis")]
+    #[serde(default, deserialize_with = "deserialize_duration_from_u64_millis")]
```

Comment:
Is there a reason these are the only fields on which you've applied the `default` attribute?

Reply:
`default` is needed when `deserialize_with` is being used to default the field to `None` if it isn't specified. I added `default` to these fields specifically because they weren't properly deserializing from the test files.

Reply:
Makes sense. Does it make more sense to include `default` as a container attribute instead now that the options structs aren't `Option`s? I guess every field is optional so it doesn't matter much either way.

Reply:
Yeah it shouldn't matter, `default` is only really necessary when we're overriding the default deserialization behavior with `deserialize_with` because the method provided there doesn't handle `Option`s.